pyproject.toml

```
@@ -8,9 +8,10 @@ authors = [
     {name = "yurirocha15", email = "yurirocha15@gmail.com"},
 ]
 dependencies = [
-    "gym>=0.26.2",
     "pygame>=2.1.0",
     "matplotlib>=3.7.1",
+    "stable-baselines3 @ git+https://github.com/DLR-RM/stable-baselines3",
```

I think it would be better to pin this to a specific commit to avoid unexpected errors due to recent commits.
This should pin it for the current latest commit.
```suggestion
    "stable-baselines3 @ git+https://github.com/DLR-RM/stable-baselines3@9cebedc89fcabf509241d89b09bf39fb61cab94b",
```